style.css

```diff
@@ -40,18 +48,115 @@ header h1 {
 
 input {
   font-size: 1.2rem;
-  max-width: 16rem;
+  width: 30rem;
   padding: 0.5rem;
+  margin: 0 auto;
 }
 
 input::placeholder {
   font-size: 1.2rem;
   padding: 1rem;
 }
 
+.add-btn-div {
+  width: 30rem;
+  margin: 0 auto;
+  display: flex;
+  justify-content: right;
+}
+
 .add-btn {
   max-width: 10rem;
   padding: 0.5rem 2rem;
   font-weight: bold;
   font-size: 1rem;
+  background-color: transparent;
+}
+
+.add-btn:hover {
+  box-shadow: 2px 2px black;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 2rem;
+  border: 2px solid black;
+  font-size: 1.2rem;
+  font-weight: bold;
+}
+
+.nav-ul {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  gap: 1rem;
+}
+
+.line {
+  height: 30px;
+  width: 2px;
+  background-color: black;
+}
+
+.hidden {
+  display: none;
+}
+
+.contact {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  max-width: 50%;
+  margin: 0 auto;
+}
+
+.contact p {
+  margin: 3rem auto;
+  font-size: 1.3rem;
+}
+
+.contact-list {
+  max-width: 50%;
+  min-width: 25%;
+  list-style-type: disc;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  font-size: 1.4rem;
+}
+
+.contact-list li {
+  margin-left: 1rem;
+}
+
+#footer {
+  position: fixed;
+  left: 2%;
+  bottom: 5%;
+  width: 95%;
+  margin: 0 auto;
+  border: 3px solid black;
+  padding: 0.5rem;
+  font-size: 1.2rem;
+  font-weight: bold;
+  display: flex;
+  gap: 2rem;
+}
```

Comment:
[OPTIONAL] There is an overlap between the content and the footer. It would be nice to fix it to improve your UI design.
Your design:
<img width="957" alt="image" src="https://user-images.githubusercontent.com/77190203/153219276-b07fa16d-7ecf-4988-b649-219a6c8e714e.png">